keps/sig-apps/3017-pod-healthy-policy-for-pdb/kep.yaml

```diff
@@ -0,0 +1,42 @@
+title: Pod Healthy Policy for PDB
+kep-number: 3017
+authors:
+  - "@mortent"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-node
+  - sig-scheduling
+status: provisional
```

Comment:
FWIW - if you want to target 1.24, you need to change to implementable and that will then force you do have PRR approved.

Reply:
I have updated and added the PRR file.